src/server/game/AI/ScriptedAI/ScriptedCreature.cpp

```diff
@@ -135,6 +135,38 @@ Creature* SummonList::GetCreatureWithEntry(uint32 entry) const
     return nullptr;
 }
 
+bool SummonList::IsAnyCreatureAlive() const
+{
+    for (StorageType::const_iterator i = storage_.begin(); i != storage_.end(); ++i)
```

Comment:
this is optional, but you could just short `StorageType::const_iterator` with an `auto`:

```suggestion
    for (auto i = storage_.begin(); i != storage_.end(); ++i)
```

so you don't have to type that long useless type

Reply:
just `auto const&`